src/main/java/org/humancellatlas/ingest/submission/SubmissionEnvelopeService.java

```diff
@@ -120,13 +109,18 @@ public SubmissionEnvelope createUpdateSubmissionEnvelope() {
         SubmissionEnvelope updateSubmissionEnvelope = new SubmissionEnvelope();
         submissionEnvelopeCreateHandler.setUuid(updateSubmissionEnvelope);
         updateSubmissionEnvelope.setIsUpdate(true);
-        SubmissionEnvelope insertedUpdateSubmissionEnvelope = submissionEnvelopeRepository.insert(updateSubmissionEnvelope);
-        submissionEnvelopeCreateHandler.handleSubmissionEnvelopeCreation(updateSubmissionEnvelope);
+        SubmissionEnvelope insertedUpdateSubmissionEnvelope = createSubmissionEnvelope(updateSubmissionEnvelope);
+        return insertedUpdateSubmissionEnvelope;
+    }
+
+    public SubmissionEnvelope createSubmissionEnvelope(SubmissionEnvelope submissionEnvelope) {
+        SubmissionEnvelope insertedUpdateSubmissionEnvelope = submissionEnvelopeRepository.insert(submissionEnvelope);
```

Comment:
`insertedUpdateSubmissionEnvelope` - does this piece of code specifically apply to update submissions? If so the method name should be updated

Reply:
do you mean reverted?

Reply:
oh i get what you mean.

Reply:
i think it should be insertedSubmissionEnvelope

Reply:
done, thanks!